src/shared/helpers/formatters/avatar.tsx

```diff
@@ -58,7 +58,7 @@ export const getAvatarProps = (
 };
 
 export const renderAvatar = (
-	profile: Avo.User.Profile | null,
+	profile: Avo.User.Profile | undefined,
```

Comment:
Dit klopt niet, als een profile niet gevonden is returned graphql null.
Als je undefined nodig hebt voor een andere reden moet je mss dit gebruiken: 
```
Avo.User.Profile | null | undefined gebruiken
```

Reply:
Not sure waarom ik er `undefined` had bijgezet maar werkt zonder so back to what it was :)